_pages/integrations/arize.md

```diff
@@ -0,0 +1,253 @@
+if not collection.exists():
+    collection.create()
+
+# Upload model file and test data to data collection.
+client.file(collection_uri+"/"+model_file).putFile(model_file_path)
+{% endhighlight %}
+
+## Generating explainability values using SHAP
+SHAP (SHapley Additive exPlanations) is a game theoretic approach to explain the output of any ML model. For in-depth details on how to use the shap library, visit [SHAP Core Explainers](https://shap-lrjball.readthedocs.io/en/docs_update/generated/shap.Explainer.html#shap-explainer). The code below creates a visual to verify that SHAP values are being properly generated for explainability (to generate the plot, you'll need to install `matplotlib`):
+
+{% highlight python %}
+import shap
+def get_shap_values(model, X_data, ExplainerType=shap.Explainer, show_graph=False):
+    # NOTE: If there are errors, you'll need to manually select the explainer class.
+    explainer = ExplainerType(model, X_data)
+    shap_values = explainer.shap_values(X_data)
+
+    # In development, it can be valuable to check graphs for feature explainability.
+    if show_graph:
+        shap.summary_plot(shap_values, X_data, feature_names=data['feature_names'])
+
+    # NOTE: The Arize API expects a pandas DataFrame of the same shape and with
+    #       the same column names as the model features.
+    return pd.DataFrame(shap_values, columns=data["feature_names"])
+
+shap_values = get_shap_values(clf, X, show_graph=True)
+{% endhighlight %}
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/arize/shap_values.png" alt="Shap values plot in Jupyter notebook" class="screenshot">
+
+The following represents the algorithm code that you would deploy on Algorithma, not the training platform used above. {:.notice-info}
+
+## Setting up your Algorithmia environment
+To begin, on Algorithmia you'll need to [create an algorithm](/developers/algorithm-development/your-first-algo) using an environment with Python 3.6 or later. In your algorithm's `requirements.txt` file, add the Arize Python library to add the monitoring capabilities provided by Arize, as well as the `shap` library and the standard ML dependencies `joblib` and `pandas`:
+
+{% highlight python %}
+algorithmia>=1.0.0,<2.0
+arize>=2.1.1
+joblib==1.0.1
+pandas<2.0,>=0.25.3
+shap==0.39.0
+{% endhighlight %}
+
+## Deploying your model on Algorithmia
+Set the environment variables `ARIZE_API_KEY` and `ARIZE_ORG_KEY` with your Arize keys; these keys; these secrets are accessible through the [Arize Settings page](https://app.arize.com/admin).
+
+Remember from above that you must also set the `ALGORITHMIA_API_KEY` environment variable with the value of your Algorithmia API key if you're running the algorithm from outside of the Algorithmia Web IDE; this API key will data-read permissions. You must also replace the `COLLECTION_OWNER` and `COLLECTION_NAME` strings with the account name and collection name where the model is stored. 
```

Comment:
```suggestion
Remember from above that you must also set the `ALGORITHMIA_API_KEY` environment variable with the value of your Algorithmia API key if you're running the algorithm from outside of the Algorithmia Web IDE; this API key will have read access only. You must also replace the `COLLECTION_OWNER` and `COLLECTION_NAME` strings with the account name and collection name where the model is stored. 
```

Reply:
changed to say "only needs to have" instead of "will have"